### DISCUSSION TITLE
Curious: How are people using Kanban & Canvases?

### DISCUSSION THREAD

**Discussion by mbbroberg** [mgmeyers/obsidian-kanban#772] [Ideas & Feature Requests]
Hey all, I'm really digging Canvas functionality, but now stuck with lots of todos across multiple places. 

Has anyone found a good way to visualize tasks and keep it in sync?

**Comment by mbbroberg** [comment]
My dream: any new card I add to my canvas is also added to the Kanban. I can either tag or have "areas" that align to my columns and I can drag/drop across them (but also add arrows to show relationships and think visually when it helps). 

Toggling back and forth would be beautiful. Not sure how complicated it is so I haven't asked for it as a feature... Would love other's thoughts!

**Reply by EggofTreachery** [reply]
I know this is old but [[Canvas Name.canvas|Task Name]] will let you link the canvas :D!
